README.md

```diff
@@ -5,17 +5,34 @@ Minio is an object storage server released under Apache License v2.0. It is comp
 Minio server is light enough to be bundled with the application stack, similar to NodeJS, Redis and MySQL.
 
 ## Docker Container
+
 ### Stable
+
+```sh
+docker run -p 9000:9000 --name minio -d minio/minio server /export
+```
```

Comment:
We can move this to `docs/docker` instead provide one single line command here.. Just like how it is done. with Binary downloads.